go.mod

```diff
@@ -22,15 +21,15 @@ require (
 	github.com/openshift/source-to-image v1.3.0
 	github.com/pkg/errors v0.9.1
 	github.com/seccomp/containers-golang v0.0.0-20190710130535-8d1ed6a872a5 // indirect
-	github.com/spf13/cobra v0.0.7
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/sys v0.0.0-20200327173247-9dae0f8f5775
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/component-base v0.18.0
+	golang.org/x/sys v0.0.0-20200622214017-ed371f2e16b4
+	k8s.io/api v0.19.0-rc.3
+	k8s.io/apimachinery v0.19.0-rc.3
+	k8s.io/client-go v0.19.0-rc.3
+	k8s.io/component-base v0.19.0-rc.3
 	k8s.io/klog v1.0.0
-	k8s.io/kubectl v0.18.0
+	k8s.io/kubectl v0.19.0-rc.3
 	k8s.io/kubernetes v1.18.0
```

Comment:
`k8s.io/kubernetes` also needs to be bumped, version tag is `v1.19.0-rc.3`. This tends to be the most difficult one.

Reply:
Indeed, takes the most effort in this part. I've updated that dependency in the last interaction, please consider.